get_gerrit_last_comment_by_id.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+command -v jq >/dev/null 2>&1 || { echo >&2 "I require jq but it's not installed. Aborting."; exit 1; }
+
+usage() { 
+    echo "Usage: $0 [-u|--gerrit_server] [-p|--port] [-i|--id] [-s|--ssh_key_path]"; 
+    exit 2;
+}
+
+declare gerrit_server port id ssh_key_path
+
+while [ $# -gt 0 ]; do
+    case $1 in
+        -g|--gerrit_server)
+            gerrit_server=$2;
+            shift
+            ;;
+        -p|--port) 
+            [[ -z $2 ]] && port=22 || port=$2;
+            shift
+            ;;
+        -i|--id) 
+            id=$2;
```

Comment:
Please remove this semicolons at the end of this and surrounding lines where these are not necessary :-)

Reply:
Change done.